### ISSUE TITLE
Version 0.8 does not contain the fix for null in serialized array

### ISSUE THREAD

**Issue by raja-baz** [Tishka17/gson-flatten#11]
It seems you released 0.8 to include the merge of my pull request, but pulling version 0.8 using gradle still ends up with the old code =/

**Comment by Tishka17** [comment]
Thank you. I'll check it and fix later this week

**Comment by raja-baz** [comment]
Yeah, it's okay, I'm pulling in my fork as a submodule for now(which is what I did to even fix the bug), but much easier to use as a dependency from gradle directly :-)

**Comment by Tishka17** [comment]
i have published version 0.8.1. Please check it.
